src/screens/MainScreen/index.js

```diff
@@ -18,16 +20,44 @@ const MainScreen = () => {
     getCalls();
   }, [currentUser]);
 
+  let testimo = null;
+  function smoothSlide(e) {
+    testimo.scrollIntoView({ behavior: "smooth" });
+  }
+
   return (
     <Container>
       {currentUser && <Title>{`Hey, ${currentUser.firstName}!`}</Title>}
-      <Subtitle>Welcome to Corona is Blind Season 3 Episode 2</Subtitle>
-      <Subtitle>This week on Corona is Blind...</Subtitle>
-      {currentUser && (
+      {currentUser && <Subtitle>Welcome to Corona is Blind Season 3 Episode 2</Subtitle>}
+      {currentUser && <Subtitle>This week on Corona is Blind...</Subtitle>}
+      {currentUser &&
         <CallListContainer>
           <CallsList callList={callList} />
         </CallListContainer>
-      )}
+      }
+      {!currentUser && <Image src={logo} />}
+      {!currentUser && <Subtitle>COLLEGE QUARANTINE EDITION</Subtitle>}
+      {!currentUser && <About>Corona is Blind: College Quarantine Edition is a1-week event designed to help college students meet one another while remaining at home during the spread of COVID-19. Based on the popular TV series “Love is Blind,” the experience consists of a series of phone and video calls between college students from all over the nation.</About>}
+      {!currentUser && <Arrow onClick={((e) => smoothSlide(e))}></Arrow>}
+      {!currentUser && <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320">
+        <path fill="#ffffff" fill-opacity="1"
+          d="M0,256L60,266.7C120,277,240,299,360,288C480,277,600,235,720,213.3C840,192,960,192,1080,192C1200,192,1320,192,1380,192L1440,192L1440,320L1380,320C1320,320,1200,320,1080,320C960,320,840,320,720,320C600,320,480,320,360,320C240,320,120,320,60,320L0,320Z">
+        </path>
+      </svg>
```

Comment:
I would suggest exporting this SVG to the assets folder instead

Reply:
done.